### DISCUSSION TITLE
I can't add, delete or modify data on the data list page。

### DISCUSSION THREAD

**Discussion by CloudWise-Lukemiao** [dbeaver/dbeaver#12728] [DBeaver Development]
I'm developing a database plug-in. I found that I can't add, delete or modify data on the data list page. How can I make it work。
![image](https://user-images.githubusercontent.com/76942485/120736058-2bd4f700-c51e-11eb-829f-500c125f12fd.png)
![image](https://user-images.githubusercontent.com/76942485/120736086-35f6f580-c51e-11eb-879b-1cf1ecae1d67.png)

**Comment by LonwoLonwo** [comment]
Hello @CloudWise-Lukemiao 
Please tell more about your database plug-in. 

Usually read-only status can be changed in Connection settings:

![2021-06-04 12_30_34-DBeaver Enterprise 21 1 0 - _dbc_ Script-28](https://user-images.githubusercontent.com/45152336/120780609-f4bc1100-c530-11eb-95ce-80bd7781f749.png)
